### ISSUE TITLE
API tokens on https://pypi.org/manage/account/ appear to be unsorted

### ISSUE THREAD

**Issue by mgedmin** [pypa/warehouse#9674]
**Describe the bug**

We use a zope.wheelbuilder account to upload binary wheels from various CI systems (Appveyor, GHA, formerly Travis).  Thus it has a bunch of API tokens (one for each project for each CI system).  These tokens appear to be listed in random order, which makes it difficult to see whether a particular project already has a token or not.

![Ekrano nuotrauka iš 2021-06-16 12-58-18](https://user-images.githubusercontent.com/159967/122199306-98e37780-cea2-11eb-9f89-23b87996ad7e.png)

**Expected behavior**

I would expect to see the tokens sorted by name. or at least by something (creation time/last usage time/scope all make sense to me).

**To Reproduce**

Create a bunch of API tokens and visit https://pypi.org/manage/account/

**Comment by mgedmin** [comment]
AFAICT this is rendered by https://github.com/pypa/warehouse/blob/14297436eed63f1dff6f91fa412a32dfad8a62d4/warehouse/templates/manage/account.html#L536-L538

which just iterates over the `orm.relationship()` defined here: https://github.com/pypa/warehouse/blob/14297436eed63f1dff6f91fa412a32dfad8a62d4/warehouse/accounts/models.py#L100-L102

SQLAlchemy's `orm.relationship()` supports [order_by=](https://docs.sqlalchemy.org/en/14/orm/relationship_api.html#sqlalchemy.orm.relationship.params.order_by), but I don't see that being used in any other relationship in the models, so maybe it would be better to do the sorting while rendering?

**Comment by di** [comment]
I agree it would be better to let the database sort these rather than the HTML renderer.

I think these should probably be sorted on creation date instead of name.
